Error for static build in arch Linux x86_64

### 

- Report crosstool-NG version : 
   HEAD: installed with brew install crosstools-ng --HEAD
- Report host OS and its version : 
 Arch Linux - x86_64 Linux 5.16.10-arch1-1 - KDE 5.91.0 / Plasma 5.24.1
- Config:
Generated config for "mipsel-unknown-linux-gnu" and some modifications
[config.txt](https://github.com/crosstool-ng/crosstool-ng/files/8101830/config.txt)

- Build.log:
[build.log](https://github.com/crosstool-ng/crosstool-ng/files/8101833/build.log)

the error for static build  :

INFO ]  =================================================================
[INFO ]  Installing binutils for host
[EXTRA]    Configuring binutils
[EXTRA]    Prepare binutils for static build
[EXTRA]    Building binutils
[ERROR]    collect2: error: ld returned 1 exit status
[ERROR]    make[5]: *** [Makefile:1041: objdump] Error 1
[ERROR]    make[5]: *** Waiting for unfinished jobs....
[ERROR]    make[4]: *** [Makefile:1431: all-recursive] Error 1
[ERROR]    make[3]: *** [Makefile:875: all] Error 2
[ERROR]    make[2]: *** [Makefile:4021: all-binutils] Error 2
[ERROR]    make[1]: *** [Makefile:1001: all] Error 2
[ERROR]   
[ERROR]  >>
[ERROR]  >>  Build failed in step 'Installing binutils for host'
[ERROR]  >>        called in step '(top-level)'
[ERROR]  >>
[ERROR]  >>  Error happened in: CT_DoExecLog[scripts/functions@376]
[ERROR]  >>        called from: do_binutils_backend[scripts/build/binutils/binutils.sh@209]
[ERROR]  >>        called from: do_binutils_for_host[scripts/build/binutils/binutils.sh@70]
[ERROR]  >>        called from: main[scripts/crosstool-NG.sh@697]
[ERROR]  >>
[ERROR]  >>  There is a list of known issues, some with workarounds, in:
[ERROR]  >>      https://crosstool-ng.github.io/docs/known-issues/
[ERROR]  >>
[ERROR]  >>  If you feel this is a bug in crosstool-NG, report it at:
[ERROR]  >>      https://github.com/crosstool-ng/crosstool-ng/issues/
[ERROR]  >>
[ERROR]  >>  Make sure your report includes all the information pertinent to this issue.
[ERROR]  >>  Read the bug reporting guidelines here:
[ERROR]  >>      http://crosstool-ng.github.io/support/
[ERROR]   
[ERROR]  (elapsed: 4:37.09)
[04:38] / make: *** [ct-ng:261: build] Error 2

--- a/scripts/build/binutils/binutils.sh
+++ b/scripts/build/binutils/binutils.sh
@@ -171,6 +171,9 @@ do_binutils_backend() {
     extra_config+=("--disable-sim")
     extra_config+=("--disable-gdb")
 
+    # libdebuginfod in incompatible with static linking
+    [ "${CT_STATIC_TOOLCHAIN}" = "y" ] && extra_config+=("--without-debuginfod")
+
     [ "${CT_TOOLCHAIN_ENABLE_NLS}" != "y" ] && extra_config+=("--disable-nls")
 
     # Disable usage of glob for higher compatibility.

